regulations/static/components/tooltips/CopyBtn.vue

```diff
@@ -0,0 +1,259 @@
+            @focusout="handleExit"
+            @mouseenter="handleEnter"
+            @mouseleave="handleExit"
+            @click="handleClick"
+        >
+            <i class="fa fa-link"></i>
+            <span v-if="btn_type === 'labeled-icon'">{{ label }}</span>
+        </button>
+        <div
+            v-show="entered && btn_type === 'icon'"
+            class="copy-tooltip hovered"
+            :class="tooltipClasses"
+            :style="tooltipStyles"
+        >
+            <p class="hover-msg">{{ label }}</p>
+        </div>
+        <div
+            v-if="clicked"
+            class="copy-tooltip clicked"
+            :class="tooltipClasses"
+            :style="tooltipStyles"
+            v-clickaway="handleCloseClick"
+        >
+            <button
+                class="close-btn text-btn"
+                aria-label="close copy link or citation dialog"
+                @click="handleCloseClick"
+            >
+                <svg
+                    width="11"
+                    height="11"
+                    viewBox="0 0 11 11"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                >
+                    <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M1.47149 1.08383L5.49969 5.11209L9.52851 1.08383C9.63637 0.975965 9.81124 0.975965 9.91911 1.08383C10.027 1.19169 10.027 1.36656 9.91911 1.47442L5.89023 5.50262L9.91911 9.53144C10.027 9.6393 10.027 9.81417 9.91911 9.92204C9.81124 10.0299 9.63637 10.0299 9.52851 9.92204L5.49969 5.89316L1.47149 9.92204C1.36363 10.0299 1.18876 10.0299 1.0809 9.92204C0.973035 9.81417 0.973035 9.6393 1.0809 9.53144L5.10916 5.50262L1.0809 1.47442C0.973035 1.36656 0.973035 1.19169 1.0809 1.08383C1.18876 0.975965 1.36363 0.975965 1.47149 1.08383Z"
+                    />
+                </svg>
+            </button>
+            <p class="citation-title">{{ this.formatted_citation }}</p>
+            <div class="action-btns">
+                <ActionBtn
+                    @action-btn-click="handleActionClick"
+                    :selectedAction="selectedAction"
+                    :status="copyStatus"
+                    actionType="link"
+                ></ActionBtn>
+                <ActionBtn
+                    @action-btn-click="handleActionClick"
+                    :selectedAction="selectedAction"
+                    :status="copyStatus"
+                    actionType="citation"
+                ></ActionBtn>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import ActionBtn from "./ActionBtn.vue";
+
+const getAnchorY = (el, elType) => {
+    if (!el) return 0;
+
+    return elType === "labeled-icon"
+        ? el.offsetWidth / 2
+        : el.offsetWidth * 0.7;
+};
+
+const getAnchorX = (el, elType) => {
+    if (!el) return 0;
+
+    return parseInt(window.getComputedStyle(el).fontSize) + 20;
```

Comment:
idk if this will reliably return an integer.  I looked at these docs and it is clearly returning 32px.  https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle